networks/3_module_validation.py

```diff
@@ -102,23 +117,24 @@ def plot_dist_modules(threshold_list):
         )
 
         fig.set_title(
-            f"Histogram of size of modules using threshold {threshold_list[i]}",
+            f"Histogram of size of modules using {clustering_method_list[i]}",
             fontsize=12,
         )
         handles, labels = fig.get_legend_handles_labels()
         fig.legend(handles, labels, loc="center right")
 
 
 # Plot distribution of modules per threshold
-plot_dist_modules(corr_threshold_list)
+plot_dist_modules(clustering_method_list)
 
 # **Takeaway:**
-# * Looks like as we decrease our correlation threshold (i.e. connections don't need to be very strong between nodes), more modules are able to form which is what I would expect.
-# * Genes in PA14 are less concentrated in one large module - there are many more small modules found.
+# * Looks like there is one large modules using DBSCAN and hierarachal clustering
+# * There are more even sized modules using affinity propogation
```

Comment:
So each distribution should show the distribution of the size of modules where the x-axis is the module id and the height of the bar is the number of genes contained within the module, but since this is a histogram we have bins which are grouping several modules together. So it would be more clear if we had each module plotted separately.